diff --git a b
index 09d0d42..7b22472 100644
--- a//Cargo.toml
+++ b//Cargo.toml
@@ -53,10 +53,6 @@ switcheroo-control = { git = "https://github.com/pop-os/dbus-settings-bindings"
 zbus = { version = "4.0", default-features = false, features = ["tokio"] }
 unicode-truncate = "2.0.0"
 unicode-width = "0.2.0"
-[patch.'https://github.com/pop-os/libcosmic']
-libcosmic = { path = "../libcosmic" }
-cosmic-config = { path = "../libcosmic/cosmic-config" }
-cosmic-theme = { path = "../libcosmic/cosmic-theme" }
 # [patch."https://github.com/pop-os/launcher/"]
 # pop-launcher = { git = "https://github.com/wiiznokes/launcher/", rev = "86a54d54a68b832d404ef83a03af84cf9ef3e694" }
 # pop-launcher-service = { git = "https://github.com/wiiznokes/launcher/", rev = "86a54d54a68b832d404ef83a03af84cf9ef3e694" }
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -1228,6 +1228,7 @@ dependencies = [
 [[package]]
 name = "cosmic-config"
 version = "0.1.0"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "atomicwrites",
  "cosmic-config-derive",
@@ -1249,6 +1250,7 @@ dependencies = [
 [[package]]
 name = "cosmic-config-derive"
 version = "0.1.0"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "quote",
  "syn 1.0.109",
@@ -1346,6 +1348,7 @@ dependencies = [
 [[package]]
 name = "cosmic-theme"
 version = "0.1.0"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "almost",
  "cosmic-config",
@@ -2701,6 +2704,7 @@ dependencies = [
 [[package]]
 name = "iced"
 version = "0.14.0-dev"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "dnd",
  "iced_accessibility",
@@ -2718,6 +2722,7 @@ dependencies = [
 [[package]]
 name = "iced_accessibility"
 version = "0.1.0"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "accesskit",
  "accesskit_winit",
@@ -2726,6 +2731,7 @@ dependencies = [
 [[package]]
 name = "iced_core"
 version = "0.14.0-dev"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "bitflags 2.9.0",
  "bytes 1.10.1",
@@ -2750,6 +2756,7 @@ dependencies = [
 [[package]]
 name = "iced_futures"
 version = "0.14.0-dev"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "futures",
  "iced_core",
@@ -2775,6 +2782,7 @@ dependencies = [
 [[package]]
 name = "iced_graphics"
 version = "0.14.0-dev"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "bitflags 2.9.0",
  "bytemuck",
@@ -2796,6 +2804,7 @@ dependencies = [
 [[package]]
 name = "iced_renderer"
 version = "0.14.0-dev"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "iced_graphics",
  "iced_tiny_skia",
@@ -2807,6 +2816,7 @@ dependencies = [
 [[package]]
 name = "iced_runtime"
 version = "0.14.0-dev"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "bytes 1.10.1",
  "cosmic-client-toolkit",
@@ -2822,6 +2832,7 @@ dependencies = [
 [[package]]
 name = "iced_tiny_skia"
 version = "0.14.0-dev"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "bytemuck",
  "cosmic-text",
@@ -2837,6 +2848,7 @@ dependencies = [
 [[package]]
 name = "iced_wgpu"
 version = "0.14.0-dev"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "as-raw-xcb-connection",
  "bitflags 2.9.0",
@@ -2867,6 +2879,7 @@ dependencies = [
 [[package]]
 name = "iced_widget"
 version = "0.14.0-dev"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "cosmic-client-toolkit",
  "dnd",
@@ -2885,6 +2898,7 @@ dependencies = [
 [[package]]
 name = "iced_winit"
 version = "0.14.0-dev"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "cosmic-client-toolkit",
  "dnd",
@@ -3354,6 +3368,7 @@ checksum = "d750af042f7ef4f724306de029d18836c26c1765a54a6a3f094cbd23a7267ffa"
 [[package]]
 name = "libcosmic"
 version = "0.1.0"
+source = "git+https://github.com/pop-os/libcosmic/#7151638f519941d8439ad0ee05723c39ed561941"
 dependencies = [
  "apply",
  "ashpd 0.9.2",
